### ISSUE TITLE
Allow users to opt in instead of pre-selecting users.

### ISSUE THREAD

**Issue by Neaox** [jolicode/secret-santa#139]
We do secret Santa as an opt-in activity at our workplace. It would be cool if people could send a message to Rodolf saying they want to join secret Santa.

**Comment by pyrech** [comment]
Hi,

Yeah I understand this would quite ease your workflow. But this is project do not have a database (mainly for privacy purposes and for simplifying everything) so we can not really have that kind of features. All the users must be manually selected before the app is run.

As a workaround, you can use Slack usergroups / Discord roles to group people before the draw. Then you can select this group directly from our app:

![image](https://user-images.githubusercontent.com/2021641/100086641-facc8900-2e4d-11eb-8e63-56dfdf8cb9ce.png)

**Comment by joellabes** [comment]
Could it instead work by looking at the members of a channel? We would announce that Secret Santa is happening, and tell everyone to join a #secret-santa channel or something. Once everyone has joined we could go into the Secret Santa app and start the pairing process.

**Comment by pyrech** [comment]
Absolutely, that's something we already had in our minds and that shouln't be too hard to implement.

If any one want to give it a try, please be my guest :wink:
